docker-compose.yml

```diff
@@ -47,6 +47,7 @@ services:
       - ct_server
   dex-idp:
     image: dexidp/dex:v2.30.0
+    user: root
```

Comment:
what if we change the location of the config file to a tmp directory?

Reply:
I tried this out too, I get permission issues with any directory. For example, if I mount ./config/dex to /tmp, the default user can't access /tmp. When I shell into the image without mounting a file, I'm able to access /tmp though, so I assume this is from the docker process that mounts the directory overriding the directory permissions.